gatsby/config/create-pages.ts

```diff
@@ -245,4 +252,11 @@ export const createPages: GatsbyNode['createPages'] = async ({
       });
     });
   });
+
+  // Create redirects
+  const redirects = result.data.allRedirect.nodes;
```

Comment:
How to protect this from failing if no redirects exists?

Reply:
`const redirects = typeof result.data.allRedirect.nodes !== 'undefined' ? result.data.allRedirect.nodes : [];` this way?

Reply:
it will fail on query already